starters/gatsby-starter-theme-workspace/gatsby-theme-minimal/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "gatsby-theme-minimal",
   "version": "1.0.0",
-  "main": "index.js",
```

Comment:
A theme (which is a plugin) needs this + we don't support workspaces yet. So can stay as-is